samples/msgext-doc-compliance-checker/env/.env.local

```diff
@@ -0,0 +1,15 @@
+# This file includes environment variables that can be committed to git. It's gitignored by default because it represents your local development environment.
+
+# Built-in environment variables
+TEAMSFX_ENV=local
+APP_NAME_SUFFIX=local
+
+# Generated during provision, you can also add your own variables.
+BOT_ID=
+TEAMS_APP_ID=
+BOT_DOMAIN=
+BOT_ENDPOINT=
+TEAMS_APP_TENANT_ID=
```

Comment:
I can't see blob storage connect or Azure Open AI service related keys here.

Reply:
As I understand it, there is no need to update the local environment file; we only need to update the .localConfigs file.